### ISSUE TITLE
Bot na VPS

### ISSUE THREAD

**Issue by brandaogabriel** [mpcabete/bombcrypto-bot#465]
Galera, eu não sei se isso ja foi discutido aqui, mas como eu faço pra ele funcionar na vps? 
Percebo que mesmo com o bot rodando, ele não tem o controle do ponteiro do mouse na vps

**Comment by andrebonatto2** [comment]
Qual vps você utiliza?

**Comment by brandaogabriel** [comment]
Estou com um windows na google cloud e na aws. Porem acredito que isso independe, já que o SO é o mesmo

**Comment by noxfork** [comment]
eu estava com uma no google cloud e estava funcionando normal.. so fechei a VPS pq precisa de um pc da nasa pra rodar esse jogo com o bot e continuar liso acessando por um acesso remoto.

**Comment by Rxeach7** [comment]
Estou com o mesmo problema, conseguiu resolver? na google cloud tambem

**Comment by fguide** [comment]
A informacao ta meio espalhada aqui, mas em resumo:

Dá pra rodar em VPS, qualquer sistema operacional
Exige recursos mesmo, o jogo é em unity e html5, nao tem muito milagre, mas uma VM com 2vcpu e 4gb de ram dá e sobra.
Windows tem problema com RDP, precisa fazer o acesso com outro software ou tentar o sugerido em #450

```
for /f "skip=1 tokens=3" %%s in ('query user %USERNAME%') do (
  %windir%\System32\tscon.exe %%s /dest:console
)
```
No #192 também tem algumas informacoes sobre VMs, performance, etc.

**Comment by brandaogabriel** [comment]
@fguide como faz pra funcionar esse comando acima? Coloquei ele em um .bat, mas nada acontece. Abre um prompt de comando, mas rapidamente ele fecha.

**Comment by fguide** [comment]
> @fguide como faz pra funcionar esse comando acima? Coloquei ele em um .bat, mas nada acontece. Abre um prompt de comando, mas rapidamente ele fecha.

Sim, a ideia é que ele vai abrir, desconectar as sessoes do RDP e manter a tela ativa e fechar, mas voce pode executar o .bat pelo CMD para ver se deu algum erro

**Comment by brandaogabriel** [comment]
O erro

![image](https://user-images.githubusercontent.com/49542836/147388149-fa723c75-8152-4670-8f31-db327d67e85a.png)

**Comment by fguide** [comment]
Tente executar com administrador o CMD, talvez seja isso, o acesso é via remote desktop?

**Comment by brandaogabriel** [comment]
Tentei como adm e deu a mesma coisa
